scylla/tests/integration/lwt_optimisation.rs

```diff
@@ -110,7 +115,7 @@ async fn if_lwt_optimisation_mark_offered_then_negotiatied_and_lwt_routed_optima
             println!("NOBODY was queried!");
         }
```

Comment:
changes in this file (apart from adding `setup_tracing()`) should be in a separate commit.